src/test/java/com/google/api/codegen/testsrc/protoannotations/library_rest_v2_gapic.yaml

```diff
@@ -0,0 +1,602 @@
+type: com.google.api.codegen.ConfigProto
+config_schema_version: 2.0.0
```

Comment:
This is a v2 gapic yaml, it should be a tiny one. All of the method-level configs should be delivered as proto annotation. I see a lot of samples config here, but it is basically deprecated at this point and should not be used for new code.

Reply:
I am not familiar with the difference, but I'm happy to make the change. Can you point to an example of what I should be using instead?